src/app/components/HoldRequest/HoldRequest.jsx

```diff
@@ -141,7 +141,7 @@ class HoldRequest extends React.Component {
           </div>
           <div className="item-summary">
             <div className="item">
-              <h2>Something wrong with your request</h2>
+              <h2>Something went wrong with your request</h2>
```

Comment:
weird. I am pretty sure I have updated this in this commit
https://github.com/NYPL-discovery/discovery-front-end/commit/f8f90376d96cb25c41bb0fe33e39a45c70809129

but after I merged development branch from development
https://github.com/NYPL-discovery/discovery-front-end/commit/d3cc8a4f881a4dc61959336b8254dbc657010ae8

It got overwritten